sap-concur/4.0/modules/ROOT/pages/index.adoc

```diff
@@ -76,32 +76,15 @@ image::sap-concur-config-design.png[config connector]
 |Authorization| A Concur access token.
 |===
 +
-. Create an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method.
-. Configure Basic Auth with your username and password.
+. Create an access token for your Concur instance. Creating an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. To create an access token, use one of the following authentication methods:
+* https://developer.concur.com/api-reference/authentication/apidoc.html#auth_grant[Authorization grant]
+* https://developer.concur.com/api-reference/authentication/apidoc.html#password_grant[Password grant]
+* https://developer.concur.com/api-reference/authentication/apidoc.html#client_credentials[Client Credentials grant]
+* https://developer.concur.com/api-reference/authentication/apidoc.html#otp_grant[One Time Password grant]
+. After obtaining the access token for your Concur instance, use the access token in the configuration for SAP Concur Connector. The following screenshot shows an example of obtaining an access token using Password grant:
```

Comment:
```suggestion
. After obtaining the access token for your Concur instance, use the access token in the configuration for SAP Concur Connector. The following screenshot shows an example of obtaining an access token using the Password grant:
```

Reply:
I know using "the" sometimes goes against Kim's advice, but I think this sounds funny when you don't use it.